build/view.js

```diff
@@ -47,12 +47,17 @@ View.prototype = {
         }
         if (target.classList.contains('show-lie')) {
             target.classList.remove('show-lie');
+            target.classList.add('show-perceived-truth');
+        } else if (target.classList.contains('show-perceived-truth')) {
+            target.classList.remove('show-perceived-truth');
         } else {
             const row = target.parentElement;
+            /* avoids assuming how users leverage the color highlighting
```

Comment:
RIght. This was there because there can be only one lie on a line. But obviously the constraint doesn't hold for truths. So ok...